app/views/public/api/v1/portals/categories/show.html.erb

```diff
@@ -36,7 +36,7 @@
             <div class="flex flex-row items-center gap-1">
               <% author = article.author %>
               <%= render "public/api/v1/portals/thumbnail", author: author, size: 5 %>
-              <span class="text-sm text-slate-600 dark:text-slate-400 font-medium flex items-center"><%= "#{I18n.t('public_portal.common.by')} #{author.name}" %></span>
+              <span class="text-sm text-slate-600 dark:text-slate-400 font-medium flex items-center"><%= "#{I18n.t('public_portal.common.by')} #{author&.name}" %></span>
```

Comment:
This block should render only if `author.name` is present

Reply:
Done https://github.com/chatwoot/chatwoot/pull/8563/commits/2f6ef88b7aba8ec3b5ac2986f2d894dd23f4fe0b